src/proxy.ts

```diff
@@ -198,25 +198,21 @@ const server = http.createServer(function (req, res) {
       target,
       secure: false,
     });
-
-    proxyApp.on("error", function (err) {
-      console.log("app>>", req.method, target + req.url);
-      res.writeHead(500, {
```

Comment:
This is crashing the proxy because we're writing to the response after it's been closed.